enrollment-server/src/main/java/com/wultra/app/docverify/zenid/provider/ZenidDocumentVerificationProvider.java

```diff
@@ -499,4 +544,29 @@ private DocumentVerificationStatus toStatus(ZenidWebInvestigateResponse.StateEnu
         }
     }
 
+    @Nullable
+    private DocumentType toDocumentType(ZenidSharedMineAllResult.DocumentRoleEnum documentRoleEnum) {
+        switch (documentRoleEnum) {
+            case DRV:
+                return DocumentType.DRIVING_LICENSE;
+            case IDC:
+                return DocumentType.ID_CARD;
+            case PAS:
+                return DocumentType.PASSPORT;
+            default:
+                return null;
+        }
+    }
+
+    private CardSide toCardSide(ZenidSharedMineAllResult.PageCodeEnum pageCodeEnum) {
+        switch (pageCodeEnum) {
```

Comment:
I also suggest to handle the `null` value of `pageCodeEnum` parameter just to be sure.

Reply:
The null check is before calling this method, but I fixed it to be more robust at this place.